### ISSUE TITLE
Pull requests status

### ISSUE THREAD

**Issue by imvucic** [KhronosGroup/SYCL_Reference#80]
We created this issue for discuss on PR in general - blocking points/force merge or other topics.

The first question is do you need some in addition to merge these two PRs : 
https://github.com/KhronosGroup/SYCL_Reference/pull/77
https://github.com/KhronosGroup/SYCL_Reference/pull/76
It's important to us to close entire Sprint 1...

The second topic based on merge of https://github.com/KhronosGroup/SYCL_Reference/pull/79...
Here we just changed files name like pre-requirement for Sprint 2 and easier work without conflict...content is old for now
It is important to have prepared asap.

Thanks,
Marko

**Comment by gmlueck** [comment]
It looks like @mkinsner merged #77 and #76.

I just approved #79.

**Comment by imvucic** [comment]
Hi @gmlueck @mkinsner,
Do you have possibility to force or speedup review process for provided PRs?
Reason is e.g [81](https://github.com/KhronosGroup/SYCL_Reference/pull/81) have dependecies for future works and would be grate to merge before continue...
Thanks in advance,
Marko

**Comment by mkinsner** [comment]
Hi @imvucic,

We're at face-to-face meetings this week which is slowing things down, but I'll make sure to review these today.

**Comment by imvucic** [comment]
Hi @mkinsner,

Should we wait review @gmlueck for final merge of https://github.com/KhronosGroup/SYCL_Reference/pull/82?

**Comment by mkinsner** [comment]
I just merged #82.  I'll go over #85 in a few hours to approve/merge.

**Comment by mkinsner** [comment]
#85 is approved and merged.  Please let us know if anything else is blocking progress.

**Comment by imvucic** [comment]
From USM we have the last one https://github.com/KhronosGroup/SYCL_Reference/pull/87, that would be grate to be merged and following our Sprint 2 plans.

**Comment by imvucic** [comment]
Hi @mkinsner @gmlueck,

We have added all Common Interface PRs and waiting review...
Also would be great to prioritize review of _New skeleton for "Data Access" chapter_ : https://github.com/KhronosGroup/SYCL_Reference/pull/91 to unblock next story's related.

BR,
Marko

**Comment by mkinsner** [comment]
I've approved and merged #91 to unblock progress on the next chapter.  Will review the common interface PRs shortly

**Comment by imvucic** [comment]
Hi @mkinsner @gmlueck 
From our perspective we should prioritize review on New skeleton for "Expressing parallelism" chapter  : https://github.com/KhronosGroup/SYCL_Reference/pull/97  
Could you please recheck?

Thanks,
Marko

**Comment by imvucic** [comment]
One more general comment...Tomorrow is last day of our Sprint 3...In order to close appropriately we should receive review on all topics from PR list : https://github.com/KhronosGroup/SYCL_Reference/pulls
We are aware of the conference allocation from your side this week, but if there is a possibility, please process

**Comment by mkinsner** [comment]
All PRs have been approved and merged now, so thanks for making all requested changes so quickly

**Comment by imvucic** [comment]
Hi @mkinsner @gmlueck 
Could you please check/review [SYCL_Reference/pulls](https://github.com/KhronosGroup/SYCL_Reference/pulls)
For complete Data access chapter we need merge of https://github.com/KhronosGroup/SYCL_Reference/pull/109.

Thanks,
Marko

**Comment by imvucic** [comment]
Hi @mkinsner,
We notice number of accumulated PRs without reviews...could you please share plans or availability for review in order that we finalize several chapters before end of 2023?

Best regards,
Marko

**Comment by mkinsner** [comment]
Hi @imvucic,

I've now either approved and merged, approved (pending merge conflict resolution), or left comments on all of the open PRs.  I'll monitor in the coming days and merge as the PRs are updated.

I'll be out of office next week and for a few days the week after, but will try to be active reviewing PRs again starting Jan 3.

**Comment by imvucic** [comment]
Hi @mkinsner @gmlueck @tomdeakin 
We believe from Intellias side provided all PRs and covered final Cycle 2, for today is planned to close last Sprint 9.

Could you please provide here final comment and ensuring that the content provided is correct and complete, and consistent with the SYCL 2020 specifications and behavior?

Thanks in advanced,
Marko

**Comment by mkinsner** [comment]
Hi @imvucic 

I'm not aware of any remaining gaps, but also won't do a detailed comparison against the SYCL 2020 specification today.  Has Intellias done a final analysis of the reference guide versus SYCL 2020, and believes that all gaps are closed (all SYCL 2020 API content is captured in the reference guide, and no SYCL 1.2.1 features which were removed in SYCL 2020 are in the reference guide)?

Thanks,
Mike

**Comment by imvucic** [comment]
Hi @mkinsner,

From Intellias we have done a final gap analysis in parallel and believes that all gaps are closed.
The question was only in the content if some smaller parts may have been overlooked, but we did everything according to the agreement and the regular closing of the 2nd cycle of gaps.
For now we will close Sprint 9, if something in addition please bring up here or comment in weekly meeting.

Thanks and Best regards,
Marko
